### ISSUE TITLE
Terminar el project detail 🥇 

### ISSUE THREAD

**Issue by JosseJuventino** [JosseJuventino/_test_projectWeb#3]
Porfi c: **a**

**Comment by German234** [comment]
@JosseJuventino Can you pls check if my view is correct?

**Comment by JosseJuventino** [comment]
No

**Comment by German234** [comment]
I reopen the issue cause you didnt want to help me :C

**Comment by JosseJuventino** [comment]
It's because you're gay :)

**Comment by German234** [comment]
:rainbow_flag: ?

**Comment by Merino951401** [comment]
that's correct

**Comment by German234** [comment]
@Merino951401 silence and go work, i dont see an update in your issue since it was created

**Comment by JosseJuventino** [comment]
@German234 silence too, go to work and do not open Rocket League, do a `console.log("Close")` and sit on my legs ;)

**Comment by Merino951401** [comment]
🏳️‍🌈 ?

**Comment by German234** [comment]
@JosseJuventino And you too, pls try to work a little bit, stop looking how to create a anvil in minecraft and try to look how to finish the proyect :)

**Comment by JosseJuventino** [comment]
No, i did a 14 components, and you? nothing, go to create a components 💯
